Makefile

```diff
@@ -33,6 +42,23 @@ init: \
 	init-apps.cwl_firehose_subscriber \
 	init-ci
 
+.PHONY: init-apps
```

Comment:
you can produce a list called 

APPS := gcp_to_cwl cwd_to_slack es_idx_manager firehost_to_es_processor cwl_firehose_subscriber

and then programmatically generate all the init/plan/install targets.

p.s. you also need to make those init-XXX targets phony targets

Reply:
thanks!